.github/workflows/test-cache-hit-miss-s3api.yaml

```diff
@@ -17,6 +17,17 @@ jobs:
               owner: context.repo.owner,
               repo: context.repo.repo,
             })).data.actions_caches.map(cache => cache.key);
+            
+            const knownKeys = [
+              "lxd-devstack-keystone-swift-20.04-10.100.115.2/24",
```

Comment:
It looks like some of these constants could be calculable, e.g., `20.04` looks suspiciously like the Ubuntu series identifier. Do you think it is worth calculating/ retrieving these from the system rather than hardcoding them? It could reduce the maintenance effort of the tests, e.g., if the tests start to run on jammy rather than focal

Reply:
The cache key schema is inside the `setup-devstack-swift` action, and there's no way to extract them unfortunately. The checks here is just to make sure that we update the keys in tests when we update the cache key schema in the action.